bin/lib/cli/builds.py

```diff
@@ -170,8 +171,11 @@ def builds_rm_old(dry_run: bool, max_age: int):
     for release in get_all_releases():
         max_builds[release.version.source] = max(release.version.number, max_builds[release.version.source])
     for release in get_all_releases():
-        if release.key in current:
+        if (release.key in current) or (get_key_counterpart(release.key) in current):
```

Comment:
I'd also extract `counterpart_key` as a variable as you're using `get_key_counterpart(release.key)` in three places in the 4 or so lines here.

Reply:
good point